lib/parser.js

```diff
@@ -383,6 +394,44 @@ function controlFlowParser(keyStart, keys)
   return statement;
 }
 
+/**
+ * Handles the ending of statements:
+ * wow [return vals]
+ * wow& [return vals]
+ */
+function handleWow_WowAnd(keys)
```

Comment:
The underscore seems strange here.  I think it would be okay to just call it handleWow